src/Uno.UI.Runtime.Skia.Wpf/Islands/UnoXamlHost.host.cs

```diff
@@ -0,0 +1,127 @@
+﻿#nullable enable
+
+using System.ComponentModel;
+using System.Windows;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using SkiaSharp;
+using Uno.UI.Runtime.Skia.Wpf;
+using Windows.Devices.Input;
+using Windows.Graphics.Display;
+using WinUI = Windows.UI.Xaml;
+using WpfControl = global::System.Windows.Controls.Control;
+
+namespace Uno.UI.XamlHost.Skia.Wpf
+{
+	/// <summary>
+	/// UnoXamlHost control hosts UWP XAML content inside the Windows Presentation Foundation
+	/// </summary>
+	abstract partial class UnoXamlHostBase : WpfControl, WinUI.ISkiaHost, IWpfHost
```

Comment:
```suggestion
	partial class UnoXamlHostBase : WpfControl, WinUI.ISkiaHost, IWpfHost
```

Also the filename does not match